nomenclature/definitions/variable/transport_types.yaml

```
@@ -0,0 +1,63 @@
+# This list of fuels will replace any occurrence of `<Transport>`
+# in a variable-definition yaml file
+
+<Transport>:
+  Maritime:
+     description: Martime transportation technologies
+  
+  Maritime|Biofuels:
+     description: Martime biofuels-based transportation technologies
+  
+  Maritime|LNG:
+     description: Martime lng-based transportation technologies
+  
+  Maritime|Oil:
+     description: Martime oil-based transportation technologies
+  
+  Rail:
+     description: Rail-based transportation technologies
+  
+  Rail|Electric:
+     description: Rail-based transportation technologies using electricity
+  
+  Rail|Oil:
+     description: Rail-based transportation technologies using oil products
+  
+  Road:
+     description: Road-based transportation technologies
+  
+  Road|Electric:
+     description: Road-based transportation technologies using electricity
+  
+  Road|Electric|BEV:
+     description: Road-based transportation technologies using battery electric vehicles
+  
+  Road|Electric|FCEV:
+     description: Road-based transportation technologies using fuel cell electric vehicles
+  
+  Road|Electric|Overhead:
+     description: Road-based transportation technologies using electricity via overhead trolley trucks
+  
+  Road|PHEV:
+     description: Road-based transportation technologies using plugin hybrid vehicles
+     
+  Road|ICE:
+     description: Road-based transportation technologies using internal combustion engines
+     
+  Road|ICE|LNG:
+     description: Road-based transportation technologies using LNG in internal combustion engines
+                  
+  Road|ICE|Oil:
+     description: Road-based transportation technologies using oil-products in internal combustion engines
+  
+  Air:
+     description: Air-based transportation technologies
+  
+  Air|Biofuels:
+     description: Air-based transportation technologies using biofuels
+  
+  Air|Hydrogen:
+     description: Air-based transportation technologies using hydrogen
+  
+  Air|Oil:
+     description: Air-based transportation technologies using oil products
```

adding an empty line at EOF
```suggestion
     description: Air-based transportation technologies using oil products

```